mlrun/data_types/spark.py

```diff
@@ -125,23 +125,26 @@ def get_df_stats_spark(df, options, num_bins=20, sample_size=None):
     summary_df.set_index(["summary"], drop=True, inplace=True)
     if summary_renamed:
         summary_df.rename(columns={"__summary_internal__": "summary"}, inplace=True)
+    # pandas df.describe() returns std, while spark returns stddev
+    # we therefore need to rename stddev to std for compatibility with pandas
+    # TODO: we may want to consider changing std to stddev in 1.2 and beyond (this requires a change to mlrun-ui)
+    summary_df.rename(index={"stddev": "std"}, inplace=True)
 
     results_dict = {}
     hist_columns = []
+    numerical_spark_types = {"int", "bigint", "float", "double", "bigdecimal"}
```

Comment:
Please also add a comment here (like the one above), explaining why we need all this mess. Specifically that the Spark `summary` returns everything in strings, and we want to convert some values to numeric both since it makes more sense and for compatibility with the Pandas implementation.
Oh, and why we omit the `50%` stat...

Reply:
I have no idea why we omit 50%.